features/html_generation.feature

```diff
@@ -2,12 +2,26 @@ Feature: html generation
 
   Lurker generates pretty info pages based on schema information
   and form which allow you to test live api.
-  When testing your api ensure that CSRF protection is handled properlya
+  When testing your api ensure that CSRF protection is handled properly
   with `protect_from_forgery :null_session`
 
   @javascript
   Scenario: json schema gets generated into html preview using "users/create"
     Given an empty directory named "public/lurker"
+    And a service file with:
+      """yml
+      ---
+      basePath: ''
+      description: ''
+      domains:
+        '[Sandbox] Heroku': 'http://lurker-app.herokuapp.com'
+        '[Real] razum2um.me': 'http://lurker.razum2um.me'
+      name: Lurker Demo Application
+      extensions: {}
+      consumes:
```

Comment:
It's `consumes`, not `consumeRs`, and it's in line with other API spec tools, like Swagger (http://swagger.io/specification/).